pennylane/hf/convert.py

```diff
@@ -0,0 +1,318 @@
+# Copyright 2018-2022 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+This module contains the functions needed for computing the molecular Hamiltonian.
+"""
+import numpy as np
+import openfermion
+import pennylane as qml
+from openfermion import QubitOperator
```

Comment:
@soranjh it's not entirely clear to me, but will this cause issues for users who `import pennylane`? if so, it might make sense to simply import `QubitOperator` only in the functions below that need it, and use a `try-except`.

Reply:
it seems that `_pennylane_to_openfermion` is the only function that needs this import

Reply:
Yep, I think it will cause an issue for the users who try to import pennylane without having openfermion installed on their machine. So we should import it separately only in the functions that need it.

Reply:
Added `try-except` for importing `openfermion`.

Reply:
:+1: This line can be deleted now!